lib/util/eyeglass_modules.js

```diff
@@ -449,19 +465,19 @@ function getFinalModule(name) {
 /**
   * resolves the module and it's dependencies
   *
-  * @param   {String} pkg - the path to the modules package.json location
+  * @param   {String} pkgPath - the path to the modules package.json location
   * @param   {Boolean} isRoot - whether or not it's the root of the project
   * @returns {Object} the resolved module definition
   */
-function resolveModule(pkg, isRoot) {
-  pkg = packageUtils.getPackage(pkg);
+function resolveModule(pkgPath, isRoot) {
+  var pkg = packageUtils.getPackage(pkgPath);
```

Comment:
`getPackage` returns an object `{ path: ..., data: ... }`

so you shouldn't need to preserve `pkgPath` here, you can reference `pkg.path`